src/Idris/Parser.hs

```diff
@@ -1569,14 +1570,14 @@ parseProg :: SyntaxInfo -> FilePath -> String -> Maybe Delta ->
 parseProg syn fname input mrk
     = do i <- getIState
          case runparser mainProg i fname input of
-            Failure doc     -> do -- FIXME: Get error location from trifecta
+            Failure ErrInfo{..}     -> do -- FIXME: Get error location from trifecta
```

Comment:
In my opinion, `RecordWildCards` makes the code difficult to read, especially when it's a record from an external dependency. It took me a while to realize where `_errDoc` is bound, and that was with the `LANGUAGE` pragma on the same commit reader!